### PR TITLE
Update the sample config with documentation on how to set the camsize and musicsize.

### PR DESCRIPTION
**Pull request by allanbank**
This updates the sample files to be the same and added verbiage on appropriate values for the camsize and musicsize. Sets the defaults to a reasonable size.

### CODE REVIEW

File: doc/teslausb_setup_variables.conf.sample [hunk 1]
```diff
@@ -18,15 +18,31 @@ export shareuser=username
...
+# Recommendations on image sizes:
+# You can use relative sizes to the available space (e.g., 10%) or explicit sizes (e.g., 16G)
+#  When using snapshots:
+#    * camsize should be a fairly small explicit value to leave room for the snapshots.
+#    * musicsize should also be an explicit size  but should be large enough to hold your music
+#      library (obviously).
+#    Using images that fill the entire available space will make snapshots inoperable.
+#    Problems have been encountered with "slow USB" from the cars that seem to correlate
+#    to backing drives that are near capacity. For that reason the system tries to keep the
+#    backing drive at least 10% available at all times.
```
Comment (marcone) [inline doc/teslausb_setup_variables.conf.sample]: I assume that by "backing drive" you mean the /backingfiles partition, however the current logic is that setup reserves a fixed 6GB regardless of sd card size.
In addition, the snapshot mechanism will delete the oldest snapshots until the free size is greater than the size of the cam_disk image.

File: doc/teslausb_setup_variables.conf.sample [hunk 2]
```diff
@@ -18,15 +18,31 @@ export shareuser=username
...
+#  When not using snapshots:
+#    * camsize should still be a fairly small explicit value. The size is determined by the number of
+#      sentry mode and saved clips you expect to have before the RPi can offload them.
+#      * A single SentryClips/SavedClips event is about 1.5GB of data.
```
Comment (marcone) [inline doc/teslausb_setup_variables.conf.sample]: My SentryClips events max out at 1.2GB. Never seen one that is 1.5GB, though I've seen smaller ones.

File: doc/teslausb_setup_variables.conf.sample [hunk 3]
```diff
@@ -18,15 +18,31 @@ export shareuser=username
...
+#      * The hour of RecentClips (the car maintains only saves about an hour) is about 9GB of data.
```
Comment (marcone) [inline doc/teslausb_setup_variables.conf.sample]: that sentence in parentheses is a little weird, and the number is a little high in my experience (6.9GB is the largest RecentClips I have from the last week)

**Review by marcone — COMMENTED** [review]
doc/OneStepSetup.md also has a section about the config file. Might need to update that too.
Also, to reduce duplication, I was thinking of either removing doc/teslausb_setup_variables.conf.sample altogether, or making it a symlink to pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample (not sure if github supports symlinks)

Are you really seeing 1.5GB Sentry events?

**Comment by allanbank** [comment]
> doc/OneStepSetup.md also has a section about the config file. Might need to update that too.
> Also, to reduce duplication, I was thinking of either removing doc/teslausb_setup_variables.conf.sample altogether, or making it a symlink to pi-gen-sources/00-teslausb-tweaks/files/teslausb_setup_variables.conf.sample (not sure if github supports symlinks)
> 
> Are you really seeing 1.5GB Sentry events?

I'll take a look at doc/OneStepSetup.md and push and update.

What do you mean by github supports symlinks? git does I am just not sure what you are looking for github to do with the symlink?

For the 1.5GB, that is the entire 10 minutes but yes 1.41 is the latest one. I recently got my FSD upgrade and I think that has increased the size of the files.

File: doc/teslausb_setup_variables.conf.sample (hunk 3 above)
Comment (allanbank) [reply doc/teslausb_setup_variables.conf.sample]: Agreed on the wording. I'll fix it.

And I just double checked. My most recent hour is 8.8GB but if I look at an hour before the FSD update it is 6.8GB. I'll update to document the difference.

File: doc/teslausb_setup_variables.conf.sample (hunk 1 above)
Comment (allanbank) [reply doc/teslausb_setup_variables.conf.sample]: The 10% was from a different change I made to keep more free space (to let the drive have some shuffling room). I'll update to the current logic.

**Comment by marcone** [comment]
What I meant with my earlier comment about github supporting links is whether the github web interface displays links as links, or as the file they link to. It appears it shows them as links, which makes having that link in the doc folder not very useful. Let's just delete it then.

**Comment by allanbank** [comment]
I removed the link. 

I don't think I ever asked if you prefer single commits and for me to force push updates or if multiple commits is OK and you do a "squash and merge". 

The force makes tracking the comments on the pull request harder but some people don't like merging...

**Comment by marcone** [comment]
I generally prefer a single commit (force push)
For small updates (e.g. fixing a typo here and there) I don't mind doing the squashing myself.

**Pull request merged**